src/rules/no-large-snapshots.ts

```diff
@@ -23,7 +24,10 @@ type RuleContext = TSESLint.RuleContext<MessageId, [RuleOptions]>;
 const reportOnViolation = (
   context: RuleContext,
   node: TSESTree.CallExpression | TSESTree.ExpressionStatement,
-  { maxSize: lineLimit = 50, whitelistedSnapshots = {} }: RuleOptions,
+  {
+    maxSize: lineLimit = 50,
+    whitelistedSnapshots = {},
+  }: Omit<RuleOptions, 'inlineMaxSize'>,
```

Comment:
```suggestion
  }: RuleOptions,
```

There's no need to omit this property, since the options we pass in still match the type.
We can always add `Omit` later if we need to :)

Reply:
Yes, I added omit because `inlineMaxSize` is not used in this function.

Reply:
True, but for an internal function like this I think it's fine - helps keep things a bit simpler and require less thinking.

I'd totally agree with you if it was an exported function, like in `utils` :)

Reply:
Ok, I removed the omit.